src/index.js

```diff
@@ -1,7 +1,6 @@
 import 'regenerator-runtime/runtime'; //async/await with Parcel
 import { App } from './app/App';
 
-const BASE_API_URL = process.env.API_BASE_URL || 'https://google.com';
-
-window.onload = () =>
-  App({ renderOn: '#movie-app', options: { apiUrl: BASE_API_URL } });
+window.onload = () => {
```

Comment:
unnecessary change, the => App({}) is shorter

Reply:
Prettier did this

Reply:
Pobably it's caused by your local config of prettier or eslint, prettier config in project doesn't have such a rule and we don't have an eslint configured :P  Minor